lib/friendly_shipping/services/ship_engine/rates_options.rb

```diff
@@ -9,26 +9,40 @@ class ShipEngine
       #
       # @attribute carriers [Array<FriendlyShipping::Carrier>] a list of the carriers we want to get IDs from.
       # @attribute service_code [String] The service code we want to get rates for.
+      # @attribute ship_date [Time] The date we want to ship on.
```

Comment:
nit: is it a date or a time?

Reply:
Good point. I'll fix.

Reply:
(But in a separate PR since the change is out of scope for this one.)